src/pages/projectSettings/CreateProjectModal.test.tsx

```diff
@@ -1,25 +1,37 @@
 import { MockedProvider, MockedResponse } from "@apollo/client/testing";
-import userEvent from "@testing-library/user-event";
 import { RenderFakeToastContext } from "context/toast/__mocks__";
 import { CREATE_PROJECT } from "gql/mutations";
-import { renderWithRouterMatch as render, screen, waitFor } from "test_utils";
+import { GET_GITHUB_ORGS } from "gql/queries";
+import {
+  renderWithRouterMatch as render,
+  screen,
+  userEvent,
+  waitFor,
+} from "test_utils";
+import { selectLGOption } from "test_utils/utils";
 import { CreateProjectModal } from "./CreateProjectModal";
 
-const defaultOwner = "existing_owner";
-const defaultRepo = "existing_repo";
+const mockedUseNavigate = jest.fn();
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useNavigate: () => mockedUseNavigate,
+}));
```

Comment:
If the goal of this mock was to assert that the modal redirected the user you can test this without mocking useNavigate. 
`renderWithRouterMatch` exposes a history object that you can use to assert on the current location. 


https://github.com/evergreen-ci/spruce/blob/620562260a9e06b7996671a45e9ab33f9f2399cf/src/components/FilterBadges/useFilterBadgeQueryParams.test.tsx#L62-L75

Reply:
My goal here was really to remove a warning:
```
console.warn
    No routes matched location "/project/new-project-name/settings"
```
We've been kinda lax w.r.t. unit test warnings thus far but I think removing them would be preferable. Using the `renderWithRouterMatch` approach doesn't have a way to add additional routes at the moment, so I went with the mock instead.

Reply:
Oh great catch you should be able to augment `renderWithRouterMatch` to support that adding the following line to the method eliminates that warning and is reusable for all tasks.
![image](https://user-images.githubusercontent.com/4605522/215556672-eacc1b80-b5a0-405a-b3d7-fd845aa9837b.png)